pandas/core/series.py

```diff
@@ -1002,36 +1002,86 @@ def _set_value(self, label, value, takeable=False):
 
     def reset_index(self, level=None, drop=False, name=None, inplace=False):
         """
-        Analogous to the :meth:`pandas.DataFrame.reset_index` function, see
-        docstring there.
+        Reset the index of the Series.
+        
+        For an Index, the index name will be used (if set), 
+        otherwise a default index or level_0 (if index is already taken) 
+        will be used.
+        For a MultiIndex, return a new Series with labeling
```

Comment:
you will get a multi-column DataFrame, with each level being turned into a column, here the levels will be named level_n if the name is None.